interface/themes/style_light.css

```diff
@@ -1506,22 +1446,21 @@ table.dataTable#therapy_groups_list tbody td{
 
 .sidebar {
   background-color: #f8f8f8;
-  font-size: 95%;
+  font-size: 0.95em;
   padding-left: 0 !important;
   padding-right: 0 !important;
 }
 
 .sidebar .submenu {
   padding-left:15px;
-  font-size:90%;
+  font-size: 0.90em;
 }
 
 .sidebar ul.mainmenu {
     padding-bottom: 55px;
 }
 
 .alert-danger {
-  font-size: 100% !important;
   background-color: #FF0000 !important;
   color: #fff !important;
 }
```

Comment:
Well, percentages can stay. Any hard typed values (Such as 14pt) will have to be changed. The goal is to get all inline styling out and into CSS